### ISSUE TITLE
Please leave the manpage uncompressed

### ISSUE THREAD

**Issue by agaida** [andrew-bibb/cmst#109]
it should be up to the package management to compress this

**Comment by andrew-bibb** [comment]
Sorry, my mistake.  I keep forgetting to take out the compressed version when I upload to git.  This is not the first time this has been asked.

**Comment by agaida** [comment]
see the pull request :) - but my point is - please leave the man pages uncompressed in the qmake file - or make sure that  the original file is not overridden - most packaging tools can and will compress the man pages automagically on install

**Comment by andrew-bibb** [comment]
Understand.  They get compressed and the uncompressed version gets deleted when I check 'make' then 'make install' on my local system.  After a build works I tend to forget to uncompress them again before I upload to git.  

I manage the git uploads with a helper script and I just modified it to make sure the manpage is uncompressed before the upload.  I've pulled my qmake project files apart working on issue #107 so I can't upload a fix right at the moment.  I also see (plus you mention) there is a PR waiting so maybe what I need to do is revert what I've been working on and start it again on a separate branch.  That way I can get this one fixed and look at the PR.
